extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl

```diff
@@ -57,26 +57,12 @@
       &nbsp;
     [% END %]
   </td>
-  <th>Name</th>
+  <th>User</th>
   <td colspan="2">
-    [% target.name || target.address.user FILTER html %]
+    [% INCLUDE bug_modal/user.html.tmpl u = target %]
```

Comment:
This looks better now 😃 One more thing I should’ve noticed earlier: the **Profile** menu item in the dropdown links to the same user profile page, which might be confusing. Is there any way to remove it?

Reply:
Yeah it was hard coded. I made a change that will allow the profile link to be hidden.